CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.23.0] - 2020-11-06
+
+### updated
+
+- Upgrading `govuk-frontend` version
+
+### Compatible with
+
+- [hmrc/hmrc-frontend v1.19.1](https://github.com/hmrc/hmrc-frontend/releases/tag/v1.19.0)
+- [hmrc/play-frontend-govuk v0.53.0](https://github.com/hmrc/play-frontend-govuk/releases/tag/v0.49.0)
+- [alphagov/govuk-frontend v3.8.1](https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0)
```

Comment:
On line 18, the version should be listed as v3.9.1 and the link updated
On line 17, the link needs updating to v0.53.0